src/appraisal.rs

```diff
@@ -146,6 +146,9 @@ impl<'de> Visitor<'de> for AppraisalVisitor {
         loop {
             if self.is_human_readable {
                 match map.next_key::<&str>()? {
+                    Some("ear.veraison.key-attestation") => {
+                        appraisal.key_attestation = Some(map.next_value::<KeyAttestation>()?)
+                    }
```

Comment:
Also need the corresponding `Some(-70002)` entry on the `else` (non human readable) arm.

Reply:
see [28ba101](https://github.com/veraison/rust-ear/pull/8/commits/28ba10181b3918c0c5c05daac872e82429a1d14a)